src/components/ConversionFileErrorModal/index.tsx

```diff
@@ -31,7 +31,7 @@ const ConversionFileErrorModal: React.FC<ConversionFileErrorModalProps> = ({
             centered
             onCancel={closeModal}
         >
-            <div>
+            <p>
                 <div className={styles.warningText}>
                     {"We're sorry, there was a problem importing your file."}
                 </div>
```

Comment:
Nit: Placing a `div` inside of a `p` element seems odd to me... Consider changing the inner div to a `span` element instead?

Reply:
Ok fair point, however the span is inline and i want this to be a block element for the styling. I could change it back to being div? 

I'm realizing I didn't really think this through all the way, but my intention in changing it from a div to a p was that it is more accessible to have p tags where there is text